hera_mc/cm_dossier.py

```diff
@@ -4,15 +4,250 @@
 # Licensed under the 2-clause BSD license.
 
 """
-This contains the HookupEntry class which serves as a "dossier" for hookup entries.
-
+This contains the Entry classes which serves as a "dossier" for part entries,
+connections entries, and hookup entries.
```

Comment:
This sounds like there should be 3 objects, but I only see 2. Am I missing something?

Reply:
You are right - I missed this after updating from removing the whole confusing PartConnection thing.